app/src/test/java/org/oppia/android/app/translation/AppLanguageResourceHandlerTest.kt

```diff
@@ -624,6 +634,7 @@ class AppLanguageResourceHandlerTest {
 
     private val TURKEY_TURKISH_LOCALE = Locale("tr", "TR")
     private val HEBREW_LOCALE = Locale("he", "US")
+    private val EGYPT_ARABIC_LOCALE = Locale("ar", "EG")
```

Comment:
This seems unused--did you intend to add a test that uses it?

Reply:
No, the unused line of code is removed.